### PR TITLE
Fix high memory usage in some rare cases

### LINKED ISSUE #76: importing a batch of photos causes large memory allocation then killed by system
Thanks for great fast image cache first.
I have done a load test of importing a batch of photos about 1000+ from system album. The app crashed when the process went about 50%. 
Viewing the crash log, I found the some info like this:
 My App <ab092f688257317aabe8cb11e3f15077>         7.253      166401                0           -         41854   50  [per-process-limit]  (frontmost) (resume)

166401* 4=665M(loaded momery)

and I have launtched the profile, capturing a screenshot like this:
image
 it seems that mmap causes the huge memory allocation then app is killed by system.
At the same time, I did not call retrieve image function to show image.
Is there a way to write a batch of photos(about 1000-5000) to image table without huge memory allocation?
My test environment: iphone6, ios8.1, xcode6.1, arm64
Thanks a lot.

### PR DESCRIPTION
**Pull request by Sega-Zero**
Pull request fixes #76.

Added few `@autorelease` surroundings to prevent high memory consumption during intense saving to cache.

### CODE REVIEW

File: FastImageCache/FastImageCache/FastImageCache/FICImageTable.m [hunk 1]
```diff
@@ -233,10 +233,6 @@ - (instancetype)initWithFormat:(FICImageFormat *)imageFormat imageCache:(FICImag
     return self;
 }
 
-- (instancetype)init {
```
Comment (mallorypaine) [inline FastImageCache/FastImageCache/FastImageCache/FICImageTable.m]: Why did you remove this `init()` implementation?

**Comment by mallorypaine** [comment]
Will merge after comment is addressed.

File: FastImageCache/FastImageCache/FastImageCache/FICImageTable.m (hunk 1 above)
Comment (Sega-Zero) [inline FastImageCache/FastImageCache/FastImageCache/FICImageTable.m]: That's weird, This line shouldn't be included in that commit. Will fix that in a moment. This line is removed in my other PR  #128

Comment (Sega-Zero) [inline FastImageCache/FastImageCache/FastImageCache/FICImageTable.m]: Returned an `init` method, sorry, my mistake.

**Pull request merged**